srcpkgs/krakenx/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'krakenx'
+pkgname=krakenx
+version=0.0.3
+revision=1
+noarch=yes
+build_style=python3-module
+hostmakedepends="python3-pip"
+depends="python3 libusb"
```

Comment:
The problem is not `libusb` ...
What you need is: https://pypi.org/project/pyusb/

Reply:
Thank you. Close and reject this for now, please.
I'm seemingly struggling to much.

Reply:
````
# Template file for 'python3-pyusb'
pkgname=python3-pyusb
version=1.0.2
revision=1
noarch=yes
wrksrc="pyusb-${version}"
build_style=python3-module
pycompile_module="usb"
hostmakedepends="python3-setuptools"
depends="python3"
short_desc="Offers easy USB devices communication in Python"
maintainer="local <root@localhost>"
license="BSD-3-Clause"
homepage="https://pypi.org/project/pyusb/"
distfiles="${PYPI_SITE}/p/pyusb/pyusb-${version}.tar.gz"
checksum=4e9b72cc4a4205ca64fbf1f3fff39a335512166c151ad103e55c8223ac147362

post_install() {
	vlicense LICENSE
}
````
That worked for me.